studies/forms.py

```diff
@@ -217,17 +217,36 @@ class StudyDesignForm(TemplatedModelForm):
 
     class Meta:
         model = Study
-        fields = []
+        fields = [
+            "has_intervention",
+            "has_observation",
+            "has_sessions",
+        ]
+        widgets = {
+            "has_intervention": forms.HiddenInput(),
+            "has_observation": forms.HiddenInput(),
+            "has_sessions": forms.HiddenInput(),
+        }
 
     def clean(self):
         """
         Check for conditional requirements:
         - at least one of the fields has to be checked
         """
         cleaned_data = super(StudyDesignForm, self).clean()
-        if not cleaned_data:
-            msg = _("Je dient minstens één van de opties te selecteren")
-            self.add_error("study_types", forms.ValidationError(msg, code="required"))
+
+        # This solution is a bit funky, but by using add_error(), it appends our
+        # error msg to a built-in required error message.
+        if not "study_types" in cleaned_data:
+            error = forms.ValidationError(
+                _("Je dient minstens één van de opties te selecteren."), code="required"
```

Comment:
I recently learned (guess from who) that you're not supposed to write "één" in Dutch if there's no chance of ambiguity. Same with  "Tijd om er één aan te maken!", by the book it shouldn't have accents.